net/qpid-proton/Portfile

```diff
@@ -19,12 +19,12 @@ homepage            https://qpid.apache.org
 
 PortGroup           github 1.0
 
-github.setup        apache qpid-proton 0.22.0
+github.setup        apache qpid-proton 0.23.0
 
 PortGroup           cmake 1.0
 
-checksums           rmd160  97f8f1920286d6841c1ee6ecc9fd6ab512b67eff \
-                    sha256  6d4683e17078361e91bc33e18d99b4c047593ab75269833aca110893910df17b
+checksums           rmd160  bb3ad84994749ded1a6bdd32b938917ff2ad15f4 \
+                    sha256  a419e76e1b45db1bda24bdbcfa71b8e20ad17a086303eb9d9009b2ac5e5a6e08
```

Comment:
Note that we now add a "size" field at the end of "checksums". Please do that on your next update.

Reply:
@pmetzger will do, thanks for letting me know!